front/app/modules/commercial/content_builder/admin/components/ContentBuilderToolbox/ToolboxItem.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+// craft
+import { useEditor, Element, UserComponent } from '@craftjs/core';
+
+// Components
+import {
+  Box,
+  colors,
+  Icon,
+  IconNames,
+} from '@citizenlab/cl2-component-library';
+import styled from 'styled-components';
+import Container from '../CraftComponents/Container';
+import Text from '../CraftComponents/Text';
+
+// Intl
+import { injectIntl } from 'utils/cl-intl';
+import { InjectedIntlProps } from 'react-intl';
+
+const ToolboxItemText = styled.p`
+  color: ${colors.adminDarkestBackground};
+  font-size: 15px;
+`;
+
+interface Props {
+  componentType: string;
+  label: ReactIntl.FormattedMessage.MessageDescriptor;
+}
+
+const ToolboxItem = ({
+  componentType,
+  label,
+  intl: { formatMessage },
+}: Props & InjectedIntlProps) => {
+  const { connectors } = useEditor();
+
+  // Assign element type
+  let itemType: UserComponent<any> | undefined;
+  let iconStyle: IconNames;
+
+  switch (componentType.toString()) {
+    case 'Container':
```

Comment:
Actually I think I can just pass these in directly via props. Will change this today.

Reply:
Indeed, you can pass the icon name as a prop - no need to pass any of the rest - just handle them in the Toolbox directly :)